Add textplot_keyness function

HaiyanLW opened the pull request:
Solves issue #785

kbenoit commented:
Looking good! A couple of things:

- looks like the words stay the same, but the bars invert, when `sort = TRUE` and `sort = FALSE`. Needs to be fixed (and manually compared to the data.frame output to be verified).
- I'd prefer the default sort for the reference words to be the highest keyness words at the bottom, not a the top of the reference group.
- do you need the two new graphics package dependencies? I could not see where they were actually used. The fewer dependencies, the better.
- Please add tests.

HaiyanLW commented:
Done
- [x] removed `sort` as it should be sorted for keyness
- [x] rearranged the order of the reference words
- [x] Removed **grid** and **gtable**-- forgot to remove them when I deleted the function using them.
- [x] Added test for `show_reference`

kbenoit commented:
Looks great @HaiyanLW! I suggest one last change:

* make the default labels for the legend not "Target" and "Reference", but rather the docnames of the categories from the dfm that are being compared. Target should always get the docname of the target row. Reference can be "Reference" if there is more than one non-target row, or the docname of the reference if it is a single row. This means that for the example plot, it would be "Trump" and "Obama" for the Target and Reference groups, respectively.
